src/ontology/cl-edit.owl

```diff
@@ -7794,6 +7794,7 @@ SubClassOf(obo:CL_0000547 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000038))
 
 # Class: obo:CL_0000548 (animal cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0000548 "A eukariotic cell that is part of some Metazoa.")
```

Comment:
Consider adding "metazoan cell" as an exact synonym to align with the definitions.